assets/js/done-task.js

```diff
@@ -0,0 +1,28 @@
+import { CLICK_EVENT, HIDE_CLASS } from "/assets/js/constants.js";
+import createButton from "/assets/js/factory/createButton.js";
+
+function doneTaskEvent(done, edit, inputElement, tools, allTasks, id) {
+  done.addEventListener(CLICK_EVENT, () => {
+    inputElement.classList.add("done");
+    allTasks[id].done = true;
+    done.classList.add(HIDE_CLASS);
+    edit.classList.add(HIDE_CLASS);
+
+    const currentDate = new Date();
+    const diffTime = currentDate.getTime() - allTasks[id].date.getTime();
+    const diffHours = diffTime / (1000 * 60 * 60);
+    const diffDays = Math.ceil(diffHours / 24);
+
+    const completeTimeButtonText =
+      diffDays === 1 ? `Completed in 1 day` : `Completed in ${diffDays} days`;
+    let completeTime = createButton(
```

Comment:
variable is not being reassigned, use `const` instead.

```suggestion
    const completeTimeBtn = createButton(
```